### PR TITLE
ICU-21078 Adding script and updating docs for CLDR jars

### PR DESCRIPTION
**Pull request by hagbard**
- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21078
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [x] Documentation is changed or added

### CODE REVIEW

File: tools/cldr/cldr-to-icu/install-cldr-jars.sh [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      less -X "${LOG_FILE}"
+    fi
+    mv -f "${LOG_FILE}" "${ROOT_DIR}/last_log.txt"
+    echo "Log file: ${ROOT_DIR}/last_log.txt"
+    exit 1
+  fi
+  echo "---- Previous command succeeded ----" >> "${LOG_FILE}"
+}
+
+# First require that we are run from the same directory as the script.
```
Comment (gnrunge) [inline tools/cldr/cldr-to-icu/install-cldr-jars.sh:43]: nit: '... we are run ...' --> '... we run ...'

**Comment by sffc** [comment]
How did this get checked in with multiple CIs failing?  Master is broken.

**Comment by sffc** [comment]
Seems just one error now on master:

```
tools/cldr/cldr-to-icu/install-cldr-jars.sh
Above files did not contain the correct copyright notice. at tools/scripts/cpysearch/cpyscan.pl line 54.
The command "perl tools/scripts/cpysearch/cpyscan.pl" exited with 255.
```

**Comment by sffc** [comment]
Also, if this is intended for ICU 67, it should be committed onto maint/maint-67, not master.  You can open a new PR to cherry-pick the commit to that branch.

**Pull request merged**